runtime/litentry/src/migration/migration.md

```diff
@@ -1,3 +1,18 @@
+# Migrate
+Under fiels P9191
+The migration including the following pallets:
+Minor pallet migration
+Bounty, Democracy, Identity, Multisig, Preimage, Proxy, Treasury, Vesting
+
+Big pallet migration:
+Balances, ParachainStaking
+ChainBridge, BridgeTransfer => AssetsHandler
+
+These migration is for the follwoing task
+https://github.com/litentry/litentry-parachain/releases/tag/v0.9.19-2
```

Comment:
Minor issue, but better to fix it. 
I think all the tag numbers are wrong: `v0.9.19-02`. Otherwise the url returns `404`.

Reply:
changed!